OpenNefia.Content/Resources/Prototypes/Elona/Buff.yml

```diff
@@ -0,0 +1,112 @@
+﻿- type: Elona.Buff
```

Comment:
Nitpick: It might be more readable to put a newline between each list entry here.

Reply:
changed in 2e3ad22